subgraphs/belt-finance/README.md

```diff
@@ -16,7 +16,7 @@ Note that Belt applies a Withdrawal and Deposit Fees based on Strategy used at t
 
 Sum across all Vaults:
 
-`(Withdraw Amount * Vault Withdraw Fee) + (Deposit Amount * Vault Deposit Fee)`
+`(Withdraw Amount * Strategy Withdraw Fee) + (Deposit Amount * Strategy Deposit Fee) + Buyback Amount`
```

Comment:
would you update the methodology version here as well as in https://github.com/messari/subgraphs#development-status (should also update subgraph version in the latter)

Reply:
Done

Reply:
Also don't forget the methodology version